de.metas.vertical.pharma/src/main/java/org/adempiere/impexp/impl/PharmaImportProductInterceptor.java

```diff
@@ -101,6 +108,11 @@ public void onImport(IImportProcess<?> process, Object importModel, Object targe
 		importPrices(iproduct);
 	}
 
+	private boolean isPharmaProductImport(final I_I_Product iproduct)
+	{
+		return !Check.isEmpty(iproduct.getPharmaProductCategory_Name(), true);
+	}
```

Comment:
why `pharma product import` is considered true when `PharmaProductCategory_Name` is set?

Reply:
and more, what means `isPharmaProductImport`?

Reply:
We have some special pharma fields, that in the regular product imports don't exist, but could crash the standard product import.
Basically, I don't want that interceptor' code,PharmaImportProductInterceptor, to run, if is a standard import product, with no pharma fields.

Reply:
got it.
But then, for clarity, i would suggest to move that code from line ~80 to 108 to a method called `onPharmaProductImported`. Call that method if `isPharmaProductImport(iproduct)`.